src/actions/CategoriesActions.ts

```diff
@@ -283,7 +283,7 @@ export const getTxActionDisplayInfo = (tx: EdgeTransaction, account: EdgeAccount
   const { assetAction, chainAction, chainAssetAction, metadata, savedAction, swapData, tokenId } = tx
   const { currencyConfig, currencyInfo } = wallet
 
-  const { displayName } = tokenId == null ? currencyInfo : currencyConfig.allTokens[tokenId] ?? { displayName: '' }
+  const displayName = tokenId == null ? currencyInfo.assetDisplayName : currencyConfig.allTokens[tokenId].displayName ?? ''
```

Comment:
The last `?? ''` is wrong, because `displayName` can't be undefined. I think you meant to do `currencyConfig.allTokens[tokenId]?.displayName ?? ''`, to handle the case there the token itself is missing.